app/components/candidate_interface/application_status_tag_component.html.erb

```diff
@@ -22,22 +22,16 @@
       classes: 'govuk-!-margin-bottom-0 govuk-!-margin-top-2',
     ) do %>
       <p>
-        Some providers allow you to defer your offer. This means that you could start your course a year later.
+        You can defer your offer which means that you could start your course a year later.
```

Comment:
This sentence feels a bit odd to me without a comma after "offer"? (might just be me though).  Could it be simplified further maybe, to avoid defining "defer"?

```suggestion
        You can defer your offer and start your course a year later.
```